index.html

```diff
@@ -27,6 +27,24 @@ <h2>Седона - небольшой городок в Аризоне,<br>
         </h2>
         <p>Рассмотрим 5 причин, по которым Седона лучше, чем Гранд Каньон!</p>
       </section>
+      <section>
+        <ol>
+          <li><h3>Настоящий городок</h3>-№1-Седона - не аттракцион для туристов, там течет своя жизнь
+            <img src="img/1-real-city.jpg" alt="Фото города" width="800" height="256">
+            <ul>
+              <li><h3>Жильё</h3>Рекомендуем пожить в настоящем мотеле, всё как в кино!</li>
```

Comment:
Здесь тоже стоит использовать параграф:
```suggestion
              <li>
                <h3>Жильё</h3>
                <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
              </li>
```